### DISCUSSION TITLE
Automate Mirth Backup

### DISCUSSION THREAD

**Discussion by rjkroll** [nextgenhealthcare/connect#5936] [Feature/Enhancement Ideas]
Hi,

I wanted to see if we could get a feature, built-into Mirth, that will automate the backup process of exporting things like a full settings backup, code templates, export all channels, etc.   

We currently have a couple of homebrew solutions, and they work, but they rely on someone updating them as Mirth connect version numbers increase and issues arise.    

It would be great to be able to go into the Settings in Mirth, select a 'Full backup' or other options, point to a directory and file format like zip and set a schedule.     And yes, everyone could code their own solution using possibly the API's or CLI, but if we can do that using those than we should just have it as an option in Mirth itself anyhow.

**Comment by pacmano1** [comment]
Wrong place @rjkroll , the issue section, not discussions.

**Comment by rjkroll** [comment]
Closed this discussion and opened a IDEA request in the appropriate section.
